.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - 53-github-actions-for-running-spotless-and-tests
+  pull_request:
+    branches:
+      - main
```

Could be only this, I think:

```suggestion
on:
  pull_request:
```

---

Perfect!